warp/src/main/java/com/schibsted/nmp/warp/components/WarpPill.kt

```diff
@@ -0,0 +1,138 @@
+ * For more info, look [here](https://warp-ds.github.io/tech-docs/components/pill/)
+ *
+ *
+ * @param modifier The modifier to be applied to the pill. Defaults to [Modifier].
+ * @param text The text to be displayed in the pill.
+ * @param onClick The callback to be invoked when the pill is clicked.
+ * @param selected Whether the pill is selected. Defaults to false.
+ * @param style The style of the pill. Defaults to [WarpPillStyle.Suggestion].
+ * @param closable Whether the pill is closable. When true, the pill will display a dismiss icon. Defaults to false.
+ * @param iconContentDescription The content description to be used for the dismiss icon.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun WarpPill(
+    modifier: Modifier = Modifier,
+    text: String,
+    onClick: () -> Unit,
+    selected: Boolean = false,
+    style: WarpPillStyle = WarpPillStyle.Suggestion,
+    closable: Boolean = false,
+    iconContentDescription: String? = null
+) {
+    CompositionLocalProvider(
+        LocalColors provides colors,
+        LocalShapes provides shapes,
+        LocalTypography provides typography,
+        LocalDimensions provides dimensions
+    ) {
+        val warpPillColors: WarpPillStyleColors = when (style) {
+            WarpPillStyle.Filter -> pillStyleFilter()
+            WarpPillStyle.Suggestion -> pillStyleSuggestion()
+        }
+
+        val pillColors = InputChipDefaults.inputChipColors(
+            containerColor = warpPillColors.background,
+            selectedContainerColor = warpPillColors.backgroundActive,
+            labelColor = warpPillColors.text,
+            selectedLabelColor = warpPillColors.text,
+            leadingIconColor = warpPillColors.icon,
+            trailingIconColor = warpPillColors.icon,
+            selectedLeadingIconColor = warpPillColors.icon,
+            selectedTrailingIconColor = warpPillColors.icon,
+        )
+
+        InputChip(
+            onClick = onClick,
+            selected = selected,
+            label = {
+                WarpText(
+                    text = text,
+                    style = WarpTextStyle.Caption,
+                    color = warpPillColors.text,
+                    modifier = Modifier
+                        .padding(vertical = dimensions.space1)
+                )
+            },
+            shape = shapes.ellipse,
+            colors = pillColors,
+            border = null,
+            modifier = modifier,
+            trailingIcon = if (closable) {
+                {
+                    Icon(
+                        imageVector = Icons.Default.Close,
+                        tint = warpPillColors.icon,
+                        contentDescription = iconContentDescription,
+                        modifier = Modifier.size(dimensions.components.pillIcon)
+                    )
+                }
+            } else null
+        )
+    }
+}
+
+enum class WarpPillStyle {
+    Filter,
+    Suggestion
```

Comment:
I feel like suggestion is not really generic enough. _Filter_ is clear to me, but I would replace "_suggestion_" everywhere with "_standard_", or something as generic. BTW, in the doc linked, the two styles arte called _standard_ and _suggestion_, which increase this confusion.
in Material 3, the chips corresponding to these pills would be _filter_ and _input_

Reply:
The naming comes from our designs in Figma, but i will forward your feedback about the naming :) 
This is already implemented and live in web so i dont think we can expect a naming change.